### PR TITLE
Feat/add chain

### PR DESCRIPTION
**Pull request by karmabadger**
This adds a few functions to frontend/src/utils/metamask.ts so to add and switch chains using pop ups.

### CODE REVIEW

**Comment by karmabadger** [comment]
a few changes/additions to .env would be required. Please check .env.example, Though, the icon urls isn't required.

**Comment by karmabadger** [comment]
image
image
image

This either adds the network to the wallet if it's not added or switches to it if it's already added. See pictures for more detail.

**Comment by karmabadger** [comment]
The button has not been styled since I have no idea how the styling works with tailwind CSS and how to contribute on that end. If someone could help on that front, it would be greatly appreciated. Thank you

File: frontend/.env.example [hunk 1]
```diff
@@ -2,6 +2,16 @@
 # See https://chainlist.org/
 REACT_APP_SUPPORTED_CHAIN_ID="0x89"
 REACT_APP_SUPPORTED_CHAIN_NAME="Polygon"
+REACT_APP_RPC_URL="https://rpc-mainnet.maticvigil.com"
+# multiple urls can be used for backups, simply separate them with a comma
+REACT_APP_RPC_URLS="https://rpc-mainnet.maticvigil.com"
+REACT_APP_BLOCK_EXPLORER_URLS="https://polygonscan.com/"
+REACT_APP_CHAIN_ICON_URLS="https://polygonscan.com/images/svg/brands/polygon.svg?v=1.3"
+REACT_APP_SUPPORTED_CHAIN_NATIVE_CURRENCY_NAME="MATIC"
```
Comment (reisr3) [inline frontend/.env.example:10]: Should these include the word `DEFAULT`? Might be easier to read.

`REACT_APP_DEFAULT_CURRENCY_NAME`

**Review by reisr3 — CHANGES_REQUESTED** [review]
Looking good! Just a couple changes and questions.

**Comment by reisr3** [comment]
Hey @catmaxi ! Any questions or pushback here? I'd love to see this get merged in - you did great work!

**Review by indeavr — COMMENTED** [review]
Changes based on @reisr3 feedback. Ready for more general testing :)

**Comment by deweller** [comment]
Closing this for now since this branch is too out-of-date.

**Pull request closed without merging**